.editorconfig

```diff
@@ -13,6 +13,10 @@ insert_final_newline = true
 indent_style = space
 indent_size = 4
 
+[*.js]
```

Comment:
I know spaces, tab, and how many of each are very personal but I felt like it might be worth the nudge to get you to use 2 spaces as that seems to be the "ember thing" these days and might lead to more contributors. Up to you. I used to use 4 but now it seems to much. :)